### ISSUE TITLE
iOS Version of DolphiniOS

### ISSUE THREAD

**Issue by mk8itra1n** [WilliamLCobb/GC4iOS#6]
Well I tested out the Cydia build of DolphiniOS and the app works fine for an early alpha. It launches correctly and the in-app rom downloader works perfect. However when I launch any game so far it goes to a black screen and the on screen controls show up then it just crashes to the home screen. Wondering what's going on if anyone can fill me in. Is it that it cannot emulate any games for now or only certain games work for now?

My Specs...
Device- iPhone 6 Plus 64GB
iOS- iOS 9.0.2 (Jailbroken)
Processor- Apple A8 Chip clocked at 1.4GHz
Bit- 64 Bit
RAM- 1GB

**Comment by ghost** [comment]
The app is in alpha stage. It is in no way 100% funtional . Don't expect it to work right for a while.

**Comment by mk8itra1n** [comment]
@pmp174 Had a feeling it wouldn't be able to launch games just yet. Even for an alpha it's pretty good. The In app rom downloader is pretty damn awesome. Guessing it will have a UI similar to iNDS when it's finished

**Comment by WilliamLCobb** [comment]
The iPhone 6 has been unable to launch due to a memory check that makes sure the app can map 1GB. I'll turn it off for the 6 and upload a build tonight

**Comment by mk8itra1n** [comment]
@WilliamLCobb Ah I see. Thanks so much bro. Will that build be uploaded to Cydia as an update to DolphiniOS or will I have to do it manually through Xcode?

**Comment by mk8itra1n** [comment]
@WilliamLCobb So I tried out your new build of DolphiniOS and looks like you've successfully fixed the crashing issue. Every game I launch no longer crashes anymore however none of them ever get past a black screen on boot. Wonder what's causing this. These are all the games and all of them have a black screen when I launch them![](http://i.imgur.com/dduycwF.jpg)![](http://i.imgur.com/ZlwG45K.jpg)

**Comment by DarknesGaming** [comment]
Seems like GC4iOS is off to a great start, especially since this is a GameCube emulator.
I was really hoping a Nintendo 64 emulator, but I guess this is worth a shot.
I can't really help much because the C programming language is so hard to learn, and Java was actually the first programming language I began to learn and study. That should be an achievement for a 15-year old like myself.

**Comment by ghost** [comment]
@WilliamLCobb The Ipad version doesn't seem to have the search button .

**Comment by WilliamLCobb** [comment]
@DarknesGaming I thought iOS had plenty of N64 emulators? Also the search button was removed when I was merging into Dolphin. It will be added back soon

**Comment by MaquinaDelMisterio** [comment]
It has retroarch, wchich workes perfectly fine on 64bits, not sure on 32bits.

**Comment by ghost** [comment]
Retro arch is a disgusting piece of software. The developers paid too much attention to function and absolutely no attention to interface or human interaction. The software is great for function and is very versatile but the developers didn't care about its appearance . There is a problem when there has to be an online tutorial video on how to set up your app.

**Comment by MaquinaDelMisterio** [comment]
Are you talking of the latest one? Because it has a really cool interface, like PS3, and you can change colors.
![image](https://cloud.githubusercontent.com/assets/4268943/16018805/b5b4d22a-316b-11e6-8dfd-497737d6cfc8.png)

**Comment by ghost** [comment]
Yes it is crap. The average user does not know what an online updater,core,load configuration, etc is. what people want is an app that directly lets you play the games without having to pay attention to specific things. Also they tried to redesign it and had absolutely no regard to the IOS design language.

**Comment by MaquinaDelMisterio** [comment]
I know at beggining its hard to understand it, but when you use it, its nice. By the way, theres no other N64 emu, so, what other chances people has. If you want to play N64 or PSX, people would have to wait for a jailbreak, use this app, create a new one or wait to Revera, which i dont see future, they are dead like dodo. Hahaha.

**Comment by nfsvsgta** [comment]
Wait how do we set it like PS3 on iOS ?

**Comment by MaquinaDelMisterio** [comment]
All you gotta do, is go to settings - Driver - Menu Driver -Xmb.
Next go to Menu - Background Gradient - select your color, mine is electric blue.

**Comment by nfsvsgta** [comment]
Thanks

**Comment by MaquinaDelMisterio** [comment]
Welcome

**Comment by Fadi5555** [comment]
We really miss @WilliamLCobb to continue his amazing work on this project.

**Comment by mk8itra1n** [comment]
Got a feeling this new year might be awesome for emulators on iOS. Got my hands on the 7 Plus and it's packing some pretty decent specs for emulation

**Comment by iOS4all** [comment]
@mk8itra1n congratulations on your iPhone 7 plus. Me too I got one like you. And the best part is I found it on iOS 10.1.1. So directly I jailbreaking it.
Anyway I'm really hope from @WilliamLCobb to get back and continue on this amazing project.
Just imagine to play your favorite gamecube games in a fullspeed. Amazing isn't it?
If he could work on this project regularly it could be just a matter of time to get a fullspeed performance. This is because of these two reasons:

1- iOS devices become more powerful like the recent one (iPhone 7/7plus).
2-the dolphin project is becoming more familiar and it already on others platforms like: Win,Mac,liunx,Android. I mean it could be easy to port some codes from there.

That's it....

**Comment by mk8itra1n** [comment]
Yep I found mine on iOS 10.1.1 also can't wait to jailbreak it and get right back into emulation!

**Comment by iOS4all** [comment]
wow it's very nice because you found it on iOS 10.1.1 because as you know iPhone 7 is pretty hard to jailbreakit and the exploit is only found it on iOS 10.1.1 only.

Yeah we're waiting for all new about emulations.

**Comment by AdamBound** [comment]
I have the same issue on GC4iOS. The build I am using is 0.13. I have it on my non`jailbroken iPhone SE and whenever I load a game, I get the controls for a second, a black screen, and then it crashes.
